tests/roots/test-build-text/table_colspan.txt

```diff
@@ -0,0 +1,7 @@
+        +-----+-----+
+        | XXX | XXX |
+        +-----+-----+
+        |     | XXX |
+        +-----+     |
+        | XXX |     |
+        +-----+-----+
```

Comment:
Why are these tables indented?

Reply:
table.txt was indented so I blindly indented mines too. Removed the indentation and it still work \o/